src/background.ts

```
@@ -262,4 +263,5 @@ async function getPermissionStatus (
     default:
       throw new Error(`unknown permission domain: ${domain}`)
   }
+  return { ...msg, host, type }
```

Biome (2.1.2)

[error] This code will never be reached ...

... because either this statement will return from the function ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)